client/src/WriteReview.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class WriteReview extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div>
+      </div>
+    );
+  }
+}
+
+export default WriteReview;
```

Comment:
All of your skeleton React files look great! Only question I have is that I noticed you have them all set up as class-based components, will all of them have state, or can some of them be functional components?